### ISSUE TITLE
The operation is insecure - Safari only

### ISSUE THREAD

**Issue by patrickomeara** [Redocly/redoc#705]
This is working fine in Firefox and Chrome

```
Something went wrong...

The operation is insecure.
Stack trace
[native code]
exports@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:84:98876
e@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:107:56885
e@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:113:28843
makeStore@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:107:60507
render@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:107:61354
Go@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:58093
Ai@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:74593
Ii@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:74939
ua@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:79607
ca@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:79261
ji@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:77773
enqueueSetState@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:68:46711
setState@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:60:1843
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:107:60981
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:107:2057
a@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:107:933
promiseReactionJob@[native code]

ReDoc Version: 2.0.0-alpha.41 
Commit: a665254
```

**Comment by RomanGotsiy** [comment]
Could you provide any details?
I can't reproduce this using Safari 12.0.1.
I go to http://rebilly.github.io/ReDoc/ and see no errors in console.

**Comment by patrickomeara** [comment]
Hi @RomanGotsiy 

I can crash the app in safari, using only 

```json
{
  "openapi" : "3.0.0",
  "info" : {
    "title" : "API",
    "description" : "Description",
    "version" : "1.0.0"
  },
  "paths" : {
  },
  "components" : {
    "schemas" : {
    }
  }
}
```

```html
<!DOCTYPE html>
<html>
<head>
  <title>Fixd API Docs</title>
  <link rel="Shortcut Icon" href="https://fixd.test/favicon.png?v=" />
  <!-- needed for adaptive design -->
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">

  <!--
	ReDoc doesn't change outer page styles
	-->
  <style>
    body {
      margin: 0;
      padding: 0;
    }
  </style>
</head>
<body>
<redoc
  spec-url='/documentation/openapi.json'
  hide-download-button
></redoc>
<script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"> </script>
</body>
</html>
```

This produces the above error. I have tried in private browser mode without any extensions.

**Comment by RmenaLopez** [comment]
Is there a solution to this? I have the exact same problem:

Something went wrong...
The operation is insecure.
Stack trace
[native code]
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:95:108229
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:109:33509
e@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:109:33510
e@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:109:82553
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:103:66222
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:103:67173
Fa@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:70125
cl@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:96962
sl@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:96887
Zs@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:93917
Zs@[native code]
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:45558
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:87:4087
Yo@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:45504
Vo@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:45439
Gs@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:90702
enqueueSetState@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:79:49224
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:71:1551
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:103:66798
https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:22:2578
a@https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:22:1422
promiseReactionJob@[native code]

ReDoc Version: 2.0.0-rc.41 
Commit: 9aa7420

Im deploying the web app using aws and im getting that error only on safari, chrome works just fine

**Comment by niklasf** [comment]
For lichess.org this issue was due to a Content-Security-Policy quirk in Safari and fixed by adjusting the header: ornicar/lila@f32941c1498e34e1fb1aea4c56c2a33fa17737b2. Is your web app served with any Content-Security-Policy headers?

**Comment by RmenaLopez** [comment]
I don't really know the answer to that, I can say that the application is running on AWS CloudFront. 
Could you tell me how to check if the app has these headers so I can provide the information?
Thanks.
